src/main.leo

```diff
@@ -59,6 +59,38 @@ program privx_exchange.aleo {
     //     };
     // }
 
```

Comment:
Should there be a function calling claim_credits, claim_token, and claim_token2 all in one?